crates/egui/src/widgets/plot/mod.rs

```diff
@@ -1488,7 +1488,7 @@ impl PreparedPlot {
                 let line_strength = remap_clamp(
                     spacing_in_points,
                     MIN_LINE_SPACING_IN_POINTS as f32..=300.0,
-                    0.0..=1.0,
+                    0.2..=1.0,
```

Comment:
On `master`, when zooming in, new grid level smoothly fade in. With this change, they now suddenly pop onto screen in a very jarring manner.

You may want to tweak the gamma curve or something else, but we should definitely fade from zero here and below